pkg/engine/defaults.go

```diff
@@ -1642,6 +1642,8 @@ func DefaultDetectors() []detectors.Detector {
 		saladcloudapikey.Scanner{},
 		boxoauth.Scanner{},
 		apimetrics.Scanner{},
+		captainDataV2.Scanner{},
```

Comment:
Where is `V1`?

Reply:
If there is no `V1` , We should start with `V1`

Reply:
> Where is `V1`?

I named them according to API versions available, there was `v2` and `v3` hence the name.

Reply:
I believe the versioning is based on detectors not the API. Correct me If I am wrong @zricethezav @abmussani 
Correct way would be to start with V1

Reply:
Done, please check @kashifkhan0771